### PR TITLE
Fix compilation by casting screenheight correctly

### PR DESCRIPTION
**Pull request by rnestler**
For more information see:
https://doc.rust-lang.org/book/casting-between-types.html#as

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -14,10 +14,9 @@ fn main() {
     let mut buffer = vec![];
     let mut cursorpos:usize = 0;
     const SPALTEN:usize = 16;
-    let mut screenheight;
 
     initscr(); //start ncursesw
-    screenheight = to_usize(getmaxy(stdscr));
+    let screenheight : usize = getmaxy(stdscr) as usize;
```
Comment (dbrgn) [inline src/main.rs:19]: But what if the screen resolution is more than 65536x65536 on a 16 bit system?!

Comment (Luz) [inline src/main.rs:19]: What if the function getmaxy() returns a value below zero?

Comment (rnestler) [inline src/main.rs:19]: I'm actually not sure what happens if `getmaxy()` returns a value <0

Comment (dbrgn) [inline src/main.rs:19]: An underflow: http://is.gd/sUjkFO

I'm not really a big fan of casting in Rust, even though the behavior is clearly defined :)

Comment (Luz) [inline src/main.rs:19]: Well getmaxy() does not look to be <0 in ncurses, so everything is ok for me

**Pull request merged**